### ISSUE TITLE
Building manylinux wheels in Travis fails randomly.

### ISSUE THREAD

**Issue by robertnishihara** [ray-project/ray#886]
The linux wheels sometimes fail to build in Travis with an error like the following.

```
+ wget --no-check-certificate http://downloads.sourceforge.net/project/boost/boost/1.60.0/boost_1_60_0.tar.gz -O /ray/src/thirdparty/boost_1_60_0.tar.gz
--2017-08-29 01:46:09--  http://downloads.sourceforge.net/project/boost/boost/1.60.0/boost_1_60_0.tar.gz
Resolving downloads.sourceforge.net... 216.34.181.59
Connecting to downloads.sourceforge.net|216.34.181.59|:80... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://versaweb.dl.sourceforge.net/project/boost/boost/1.60.0/boost_1_60_0.tar.gz [following]
--2017-08-29 01:46:09--  https://versaweb.dl.sourceforge.net/project/boost/boost/1.60.0/boost_1_60_0.tar.gz
Resolving versaweb.dl.sourceforge.net... 104.238.205.251
Connecting to versaweb.dl.sourceforge.net|104.238.205.251|:443... connected.
OpenSSL: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure
Unable to establish SSL connection.
```

We should try upgrading `wget`.

**Comment by pcmoritz** [comment]
Or use curl or do retries

**Comment by xhochy** [comment]
Use retries, I also often see these errors downloading Boost from Sourceforge. This does not happen so often in Arrow as we have a base Docker image that already contains a built Boost. So the Boost download only happens, when we actually change the build script for Boost.

**Comment by robertnishihara** [comment]
Fixed by #927.
